en/deploy-on-azure-aks.md

```diff
@@ -0,0 +1,579 @@
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+kubectl apply -f tidb-cluster.yaml -n tidb-cluster && \
+kubectl apply -f tidb-monitor.yaml -n tidb-cluster
+```
+
+After the yaml file above is applied to the Kubernetes cluster, TiDB Operator creates the desired TiDB cluster and its monitoring component according to the yaml file.
+
+### View the cluster status
+
+To view the status of the starting TiDB cluster, run the following command:
+
+{{< copyable "shell-regular" >}}
+
+``` shell
+kubectl get pods -n tidb-cluster
+```
+
+When all the Pods are in the `Running` or `Ready` state, the TiDB cluster is successfully started. For example:
+
+```
+NAME                              READY   STATUS    RESTARTS   AGE
+tidb-discovery-5cb8474d89-n8cxk   1/1     Running   0          47h
+tidb-monitor-6fbcc68669-dsjlc     3/3     Running   0          47h
+tidb-pd-0                         1/1     Running   0          47h
+tidb-pd-1                         1/1     Running   0          46h
+tidb-pd-2                         1/1     Running   0          46h
+tidb-tidb-0                       2/2     Running   0          47h
+tidb-tidb-1                       2/2     Running   0          46h
+tidb-tikv-0                       1/1     Running   0          47h
+tidb-tikv-1                       1/1     Running   0          47h
+tidb-tikv-2                       1/1     Running   0          47h
+```
+
+## Access the database
+
+After you have deployed a TiDB cluster, you can access the TiDB database to test or develop your application.
+
+### SSH to AKS node
+
+The LoadBalancer created for your TiDB cluster is an intranet LoadBalancer. You can create a [Bastion](https://docs.microsoft.com/en-us/azure/bastion/tutorial-create-host-portal) in the cluster virtual network to connect to a internal host and then access the database. You can also [Create the SSH connection to a Linux node](https://docs.microsoft.com/en-us/azure/aks/ssh#create-the-ssh-connection-to-a-linux-node) and SSH to the internal host with your authentication or you can simply use tools like [node-shell](https://github.com/kvaps/kubectl-node-shell) and then access the database.
+
+> **Note:**
+>
+> In addition to the bastion host, you can also connect an existing host to the cluster virtual network by [Peering](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-network-peering-overview). If the AKS cluster is created in an existing virtual network, you can use the host in the virtual network.
+
+### Install the MySQL client and connect
+
+After SSH to the internal host, you can access the TiDB cluster via the MySQL client.
+
+1. Install the MySQL client on the host:
+
+    {{< copyable "shell-regular" >}}
+
+    ``` shell
+    sudo yum install mysql -y
+    ```
+
+2. Connect the client to the TiDB cluster:
+
+    {{< copyable "shell-regular" >}}
+
+    ``` shell
+    mysql -h ${tidb-lb-ip} -P 4000 -u root
```

Comment:
```suggestion
    mysql --comments -h ${tidb-lb-ip} -P 4000 -u root
```

Reply:
why we need --comments here ?

Reply:
With TIDB in general `--comments` is needed. This is because without that option comments are stripped before they are sent to the server. This causes problems as TiDB specific code like `/*T![clustered_index] CLUSTERED */` for TiDB and TiDB feature specific parts of statements. This is used by `SHOW CREATE TABLE...` etc.

While it might not be strictly required here, it would be good to add it anyway as it is a best practice to use `--comments` for all TiDB sessions as forgetting this might cause statements to *silently* behave differently which can be very frustrating for users and cause all kinds of issues. And you never know what a user might use the session for after doing this step of the docs.

Reply:
I see,  I reference the doc https://docs.pingcap.com/tidb-in-kubernetes/stable/deploy-on-aws-eks#install-the-mysql-client-and-connect and I think it is better to maintain the same doc format now.   
The root cause that we need `--comments` is because the difference behavior between Mysql like code  `/*T![clustered_index] CLUSTERED */` ,  so I think it need a global talk about if we need add `--comments` and then fix all related docs once.

Reply:
So let's at least do the right command in newly added doc.